CHANGELOG.md

```
@@ -1,3 +1,11 @@
+# [0.17.1](https://github.com/go-webauthn/webauthn/compare/v0.17.1...v0.17.0) (2026-05-03)
+
+
+### Bug Fixes
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)